app/build.gradle

```diff
@@ -49,6 +55,7 @@ dependencies {
     implementation generalDependencies.appcompat
     implementation generalDependencies.material
     implementation generalDependencies.constraintlayout
+    implementation 'androidx.lifecycle:lifecycle-viewmodel'
```

Comment:
if you will add a dependency, please add it in the general dependencies file

Reply:
done